src/plugins/discover/public/application/components/skip_bottom_button/skip_bottom_button.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { EuiButton } from '@elastic/eui';
+import { FormattedMessage, I18nProvider } from '@kbn/i18n/react';
+
+export interface SkipBottomButtonProps {
+  /**
+   * Action to perform on click
+   */
+  onClick: () => void;
+}
+
+export function SkipBottomButton({ onClick }: SkipBottomButtonProps) {
+  return (
+    <I18nProvider>
+      <EuiButton onClick={onClick} iconType="arrowDown" className="dscSkipButton">
+        <FormattedMessage id="discover.skipToBottomButtonLabel" defaultMessage="Skip to bottom" />
+      </EuiButton>
```

Comment:
EUI actually has an [EuiSkipLink](https://elastic.github.io/eui/#/utilities/accessibility) precisely for this use case that would be useful here without needing the extra CSS to push it off screen as EuiSkipLink will automatically hide/show when tabbing to it. I would also make the text more descriptive than "bottom".

```suggestion
      <EuiSkipLink size="s" onClick={onClick} className="dscSkipButton" destinationId="">
        <FormattedMessage id="discover.skipToBottomButtonLabel" defaultMessage="Skip to table" />
      </EuiSkipLink>
```

Reply:
Sounds good. I looked in the button section without searching for `Skip` effectively in the doc. Good suggestion, I'm going to use it.

Reply:
nice, didn't know this component 👍